README.md

```diff
@@ -1 +1,83 @@
-# Lazy
\ No newline at end of file
+# lazy
+Lazy is a generic asset lazy loader. Assets will load right before they scroll into the viewport. By default, Lazy will lazy load images, but it can also be configured to fire a generic callback whenever any element enters the viewport, making it useful for loading or animating practically any element or asset type. 
+
+## Install
+Lazy is an ES6 module. Consequently, you'll need an ES6 transpiler ([Babel](https://babeljs.io) is a nice one) and a module loader as part of your Javascript workflow.
+
+If you're already using the [JSPM package manager](http://jspm.io) for your project, you can install moduleLoader with the following command:
+
+```
+$ jspm install github:DEGJS/lazy
+```
+
+## Usage
+Sample Javascript:
+```js
+import lazy from "DEGJS/lazy";
```

Comment:
import lazy from "DEGJS/lazy.js";

If we're not using JSPM, we can't be lazy with our import paths any more. Ha, lazy.